configuration/nginx/http.conf

```diff
@@ -28,7 +28,7 @@ server {
 
     # rate limiting for django admin panel
     location ^~/admin {
-        limit_req zone=adminlimit
+        limit_req zone=adminlimit;
```

Comment:
I appreciated your fast fix but really...last PR you convinced me but I thought you did run the project just at least one time before making the PR. Just pushing code in the configuration without even trying to execute the app...well not good

Reply:
I did run it. I even tested the rate-limiting by sending repeated Postman requests. That's the weird part. I really can't say much except that maybe I pressed an accidental backspace after building the container and before making the commit. But, I understand it was a major mistake on my part. Won't be repeated again.

Reply:
This is in no way in defence of the mistake, but should we look into making tests to ensure that problems like this are caught automatically? As, in this case, the problem was quite easy to detect using tests as the container itself gave a "failure to start".

Reply:
Ok I believe that. Typos are the worst thing ever and they are difficult to catch.

Reply:
You are completely right. Tests are the way. But it seems that the CI did not notice it.

Reply:
ah I understood why. This is because you did not change the `local.conf`. That is the nginx config used by the CI

Reply:
One thing could be to extract the code you made for nginx in a separated nginx file and import it in the other ones (local,http,https) so we just have a single place with that code and it is correctly run by the CI.

That is a suggestion I should have made you beforehand but that's it. We learn from experience

Reply:
Yeah. Even I was thinking of what the ideal solution to the copy-paste would be, but couldn't come up with a good one